frontend/website/src/app.html

```diff
@@ -4,9 +4,17 @@
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.svg" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
+
+		<!-- Twitter -->
+		<meta name="twitter:card" content="summary" />
+		<meta name="twitter:title" content="Scuffle" />
+		<meta name="twitter:description" content="Scuffle - open-source live-streaming platform" />
+		<!-- TODO: Change when we don't get twitter handle -->
+		<meta name="twitter:site" content="@scuffle" />
+
 		%sveltekit.head%
 	</head>
 	<body data-sveltekit-preload-data="hover">
-		<div style="display: contents">%sveltekit.body%</div>
+		%sveltekit.body%
```

Comment:
Is this correct? I believe svelte recommends to have a div wrap around the generated code?

Reply:
I know it was in the template like this but I couldn't find any reason for the div. Do you know why it's there?